nncf/tensorflow/quantization/algorithm.py

```diff
@@ -153,6 +155,7 @@ class TFQuantizationSetup:
     def __init__(self):
         super().__init__()
         self._quantization_points = []  # type: List[TFQuantizationPoint]
+        self._unified_scale_groups = []  # type: List[List[QuantizationPointId]]
```

Comment:
Could you confirm that these changes are backward compatible and do not require updating checkpoints?

Reply:
After these changes, checkpoints without unified groups can not be loaded
```
  File "nncf/nncf/tensorflow/quantization/algorithm.py", line 275, in _load_state_without_name
    self._quantizer_setup = TFQuantizationSetup.from_state(quantizer_setup_state)
  File "nncf/nncf/tensorflow/quantization/algorithm.py", line 221, in from_state
    for quantization_group in state[cls._state_names.UNIFIED_SCALE_GROUPS]:
KeyError: 'unified_scale_groups'

```

Reply:
Fixed